metricflow/test/sql/test_engine_specific_rendering.py

```diff
@@ -106,39 +106,39 @@ def test_percentile_expr(
     sql_client: SqlClient,
 ) -> None:
     """Tests rendering of the percentile expression in a query."""
-
-    select_columns = [
-        SqlSelectColumn(
-            expr=SqlPercentileExpression(
-                order_by_arg=SqlColumnReferenceExpression(SqlColumnReference("a", "col0")),
-                percentile_args=SqlPercentileExpressionArgument(
-                    percentile=0.5, function_type=SqlPercentileFunctionType.CONTINUOUS
+    if sql_client.sql_engine_attributes.percentile_aggregation_supported:
```

Comment:
Let's mark this to skip instead. So it'd be something like:

```
if not sql_client.sql_engine_attributes.percentile_aggregation_supported:
    pytest.skip("Warehouse does not support percentile expressions")

select_columns = [
# .... rest of test here
```

Then we get a skip notice in the test runner rather than a spurious pass.